templates/patterns/inpage_navigation/pattern-inpage-navigation.html.twig

```diff
@@ -4,12 +4,14 @@
  * Inpage navigation pattern.
  */
 #}
-{{ attach_library('oe_bootstrap_theme/inpage_navigation') }}
-
+{% if links is not null %}
```

Comment:
@escuriola As mentioned, please check if links not empty, not just not null.
We don't want to show them for empty array.